scripts/update/Updater.php

```diff
@@ -728,5 +729,39 @@ public function update($initialVersion)
         }
 
         $this->skip('2.17.9', '4.1.0');
+
+        if ($this->isVersion('4.1.0')) {
+            // Update client_lib_config_registry.conf.php
+            ClientLibConfigRegistry::getRegistry()
+                ->register('taoClientDiagnostic/component/diagnostic/diagnosticLoader', [
+                    'diagnostics' => [
+                        'taoClientDiagnostic/tools/diagnostic/diagnostic' => [
```

Comment:
Sorry for the late change, but it should be better to only use the config key here: `diagnostic`.